src/modules/user/auth/sagas/logoutSaga.ts

```diff
@@ -16,6 +16,7 @@ export function* logoutSaga(action: LogoutFetch) {
     try {
         yield call(API.delete(requestOptions), '/identity/sessions');
         yield put(userReset());
+        sessionStorage.removeItem('csrfToken');
```

Comment:
A CSRF token can be included in the <meta> tag as shown below. All subsequent calls in the page can extract the CSRF token from this <meta> tag. It can also be stored in a JS variable / anywhere on the DOM. However, it is not recommended to store it in cookies or browser local storage.